Binary files orig/RPostgreSQL/src/.Makevars.win.swp and patched/RPostgreSQL/src/.Makevars.win.swp differ
diff -Nru orig/RPostgreSQL/src/Makevars.win patched/RPostgreSQL/src/Makevars.win
--- orig/RPostgreSQL/src/Makevars.win	2023-12-15 03:35:58.000000000 +0100
+++ patched/RPostgreSQL/src/Makevars.win	2025-03-26 18:31:28.696306200 +0100
@@ -1,11 +1,8 @@
-PKG_CPPFLAGS=-I./libpq
-PKG_LIBS=libpq/libpq.a -lshfolder -lwsock32 -lws2_32 -lsecur32
+ifeq (,$(shell pkg-config --version 2>/dev/null))
+  PKG_LIBS = -lpq -lpgcommon -lpgport -lssl -lcrypto -lm -lz -lsecur32 -lws2_32 -lwldap32 \
+             -lshell32 -lcrypt32 -lgdi32
+else
+  PKG_CPPFLAGS = $(shell pkg-config --cflags libpq)
+  PKG_LIBS = $(shell pkg-config --libs libpq)
+endif
 
-.PHONY: all 
-all:  $(SHLIB)
-$(SHLIB): libpq/libpq.a
-
-export CC
-
-libpq/libpq.a:
-	(cd libpq; make CUSTOM_CC="$(CC)" -f "$(R_HOME)/etc/$(R_ARCH)/Makeconf" -f Makefile.win)
